### PR TITLE
eng-899: remove tokens from samples and readme's

### PR DESCRIPTION
**Pull request by pashafateev**
Added context where relevant. Didn't touch purrr

### CODE REVIEW

File: twilio/README.md [hunk 1]
```diff
@@ -21,18 +21,11 @@ environment in the [`autokitteh.yaml`](./autokitteh.yaml) manifest file.
...
+2. Via the `ak` CLI tool, or the AutoKitteh VS Code extension, deploy the
    `autokitteh.yaml` manifest file
 
 ## Connection Notes
 
 AutoKitteh supports connecting to Twilio using either an auth token or an
-[API key](https://www.twilio.com/docs/glossary/what-is-an-api-key), which are
-configured in step 3 above.
+[API key](https://www.twilio.com/docs/glossary/what-is-an-api-key), which can
+configured in the AutoKitteh WebUI.
```
Comment (daabr) [inline twilio/README.md:31]:
```suggestion
be configured in the AutoKitteh WebUI.
```

**Review by daabr — COMMENTED** [review]
Various tiny tweaks - will approve after they're done.

File: github/autokitteh.yaml [hunk 2]
```diff
@@ -1,10 +1,6 @@
 # This YAML file is a declarative manifest that describes the setup
 # of an AutoKitteh sample project that demonstrates bidirectional
 # integration with GitHub (https://github.com).
```
[the remark this answers is not in the document]
Comment (pashafateev) [reply github/autokitteh.yaml:3]: Done

File: google/gmail/autokitteh.yaml [hunk 3]
```diff
@@ -1,10 +1,6 @@
 # This YAML file is a declarative manifest that describes the setup
 # of an AutoKitteh sample project that demonstrates integration with
 # Gmail (https://www.google.com/gmail/about/).
```
[the remark this answers is not in the document]
Comment (pashafateev) [reply google/gmail/autokitteh.yaml:3]: Done

File: google/sheets/autokitteh.yaml [hunk 4]
```diff
@@ -1,10 +1,6 @@
 # This YAML file is a declarative manifest that describes the setup
 # of an AutoKitteh sample project that demonstrates integration with
 # Google Sheets (https://www.google.com/sheets/about/).
```
[the remark this answers is not in the document]
Comment (pashafateev) [reply google/sheets/autokitteh.yaml:3]: Done

**Review by daabr — APPROVED** [review]

**Pull request merged**